cmd/template/cluster/flag.go

```diff
@@ -252,6 +254,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.PodsCIDR, flagPodsCIDR, "", "CIDR used for the pods.")
 	cmd.Flags().StringVar(&f.Release, flagRelease, "", "Workload cluster release.")
 	cmd.Flags().StringSliceVar(&f.Label, flagLabel, nil, "Workload cluster label.")
+	cmd.Flags().StringVar(&f.ServicePriority, flagServicePriority, "highest", fmt.Sprintf("Service priority of the cluster. Must be one of %v", getServicePriorities()))
```

Comment:
Can we use `label.ServicePriorityHighest` here instead of the string literal?

Reply:
Oh yeah, sure :)